code/game/objects/items/inducer.dm

```diff
@@ -172,12 +172,16 @@
 		return
 	. += "inducer-[cell ? "bat" : "nobat"]"
 
+/obj/item/inducer/empty
+	cell_type = null
+	opened = TRUE
+
+
 /obj/item/inducer/sci
 	icon_state = "inducer-sci"
 	inhand_icon_state = "inducer-sci"
 	desc = "A tool for inductively charging internal power cells. This one has a science color scheme, and is less potent than its engineering counterpart."
 	cell_type = null
-	powertransfer = 500
```

Comment:
What's this change for?

Reply:
I argued for this. https://github.com/tgstation/tgstation/pull/80126#issuecomment-1839427690

It should have been reflected in the changelog however.

Reply:
I didn't see it, but I'll respond anyway

> Probably some kind of bikeshedding design problem in the past. But I can imagine people not being aware of this discrepancy since it isn't really explained very well

``This one has a science color scheme, and is less potent than its engineering counterpart.`` is in the description of the science inducer and I don't see this as not being explained well. How can this possibly be explained better?

Reply:
Be honest with me. Did you know this was how it was designed before? Did you hear anyone else say as much? Because the only reason i know is because I made a feature that involved an inducer.

They're not even named differently, John, why would someone suspect they're different beyond a color scheme change?

Reply:
The reason I knew is because I examined the inducer and saw its description, and I think many people have before too

Reply:
This is like the inverse of what should happen, if you are making engineering ones printable they need to reflect the power of other printed ones, NOT bringing the printable ones up to power with the engineering ones.

Also this is a change unreflected in title or desc so hitting it with DNM until fixed.

Reply:
> This is like the inverse of what should happen, if you are making engineering ones printable they need to reflect the power of other printed ones, NOT bringing the printable ones up to power with the engineering ones.
> 
> Also this is a change unreflected in title or desc so hitting it with DNM until fixed.

Sorry about that, I had finals all of last week so wasnt really in the right situation to work on a PR for this. That said, would there be a problem with making there be a second engineering inducer, like say a deluxe model? That way, Engi can still have a powerful inducer, but sci cant just print crossbow parts.

Reply:
im personally fine with that